packages/commonwealth/client/scripts/views/components/GatingGrowl/GatingGrowl.tsx

```diff
@@ -0,0 +1,98 @@
+import { useCommonNavigate } from 'navigation/helpers';
+import React from 'react';
+import Persmissions from 'utils/Permissions';
+import { CWCheckbox } from 'views/components/component_kit/cw_checkbox';
+import { CWGrowl } from 'views/components/component_kit/cw_growl';
+import { CWIcon } from 'views/components/component_kit/cw_icons/cw_icon';
+import { CWText } from 'views/components/component_kit/cw_text';
+import { CWButton } from 'views/components/component_kit/new_designs/cw_button';
+import './GatingGrowl.scss';
+import { GatingImg } from './GatingGrowlImg';
+
+const GatingGrowl = () => {
+  const navigate = useCommonNavigate();
+
+  const [dontShowAgain, setDontShowAgain] = React.useState(false);
+  const [disabled, setIsDisabled] = React.useState(
+    localStorage.getItem('dontShowGatingGrowl') === 'true',
+  );
+  const isAdmin = Persmissions.isCommunityAdmin();
+
+  const handleExit = () => {
+    setIsDisabled(true);
+    if (dontShowAgain) {
+      localStorage.setItem('dontShowGatingGrowl', 'true');
+    }
+  };
+
+  return (
+    <CWGrowl disabled={disabled} position="bottom-right">
+      <div className="GatingGrowl">
+        <div onClick={handleExit}>
+          <GatingImg />
```

Comment:
This was attempted, I originally wrapped the X part of the SVG with an onclick call but that is what caused the bug that Malik found. Since the X is apart of the SVG not a separate part. 

Whats probably better here is to take the X out of the SVG all together and put a `CWIcon` instead.